jest.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    verbose: true,
+    moduleFileExtensions: ['js', 'json', 'ts', 'tsx'],
+    rootDir: '.',
+    testMatch: ['<rootDir>/src/**/*.test.ts?(x)'],
```

Comment:
Is this regex or smth similar? Which files match `.ts?(x)` extension?

Reply:
ts, tsx

Reply:
we can remove this line, because jest default is `[ "**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)" ]`